### DISCUSSION TITLE
AGENT WSERR: RangeError: Invalid WebSocket frame: MASK must be set

### DISCUSSION THREAD

**Discussion by kameleon25** [Ylianst/MeshCentral#5460] [General]
I am looking through logs and I am getting flooded with:
` AGENT WSERR: RangeError: Invalid WebSocket frame: MASK must be set`

A quick search and I can't find anything about this exact error. Am I missing something? It is repeating every few seconds.

**Comment by si458** [comment]
plz check issues including closed issue first 
https://github.com/Ylianst/MeshCentral/issues/5446
